frontend/appflowy_flutter/lib/startup/deps_resolver.dart

```diff
@@ -191,6 +192,8 @@ void _resolveHomeDeps(GetIt getIt) {
   getIt.registerSingleton<ReminderBloc>(ReminderBloc());
 
   getIt.registerSingleton<RenameViewBloc>(RenameViewBloc(PopoverController()));
+
+  getIt.registerSingleton<ViewInfoBloc>(ViewInfoBloc());
```

Comment:
If the bloc isn't using any dependencies, what's the difference between injecting it and using it directly?